gravitee-am-ui/src/assets/css/main.css

```diff
@@ -238,6 +241,11 @@ form div {
     flex-grow: 0;
 }
 
+.forgot-password {
+    font-size: 14px;
+    text-decoration: underline
+}
```

Comment:
Is it only specific for this link or for every links ?

Reply:
I created a specific class for forgot passwprd to avoid the `underline` attribute on SocialIDP button & Webauthn signin tat are currently managed by `a` element

Reply:
We should also apply it on the link to access alternative challenge.